internal/response-consumer/handler.go

```diff
@@ -168,7 +168,7 @@ func (this *handler) onMessage(ctx context.Context, msg *k.Message) {
 	}
 }
 
-func satAssignmentWithCase(responseFull bool, updateHost db.RunHost) map[string]interface{} {
+func satAssignmentWithCase(ctx context.Context, responseFull bool, updateHost db.RunHost) map[string]interface{} {
```

Comment:
not needed?

Reply:
@jharting I believe I added the ctx here for the probe below it.

Reply:
@aleccohan on what line?

Reply:
Ah!  Sorry you're right, removing!